### DISCUSSION TITLE
Blurry in different versions

### DISCUSSION THREAD

**Discussion by dongsiDaengsi** [cefsharp/CefSharp#4565] [General]
Hello everyone.
I noticed the following: In a previous version (79.1.360) the website I was looking at was displayed correctly. After updating CefSharp to the latest version 115.3.110, the website appears blurry in some places.

Attached you will find the corresponding screenshots.

**Version 79.1.360**
![79 1 360](https://github.com/cefsharp/CefSharp/assets/62557519/96eefd5e-bc72-478b-93d1-2a1f3c80f6be)

**Version 115.3.110**
![115 3 110](https://github.com/cefsharp/CefSharp/assets/62557519/c00d474c-2cdd-424c-98fb-bc18a5fabd45)

Does anyone have an idea?

**Comment by amaitland** [comment]
Does the problem reproduce with the `MinimalExample`?

Please answer the questions listed at https://github.com/cefsharp/CefSharp/wiki/Questions-&-Support#when-asking-a-question-please-include-the-following

**Comment by dongsiDaengsi** [comment]
Yes, the error can be reproduced with the MinimalExample and the effect also exists here.

![2023-08-10 10_41_47-CefSharp MinimalExample Wpf (Debug) - atvise - next generation](https://github.com/cefsharp/CefSharp/assets/62557519/e8a069ea-fa22-40d1-b447-985b13fe2fc5)

**Comment by amaitland** [comment]
- How does `Chrome` at the same version behave?
- Have you tested on a different machine?
- How does the `CEF Sample Application` behave? https://github.com/cefsharp/CefSharp/blob/master/CONTRIBUTING.md#cefsharp-vs-chromium-embedded-frameworkcef

> Please answer the questions listed at https://github.com/cefsharp/CefSharp/wiki/Questions-&-Support#when-asking-a-question-please-include-the-following

In future please review and answer any relevant questions, saves me having to retype things. Are you using `WPF` or `WinForms`?

> Yes, the error can be reproduced with the MinimalExample and the effect also exists here.

The image doesn't really look blurry to me.

**Comment by dongsiDaengsi** [comment]
I installed the latest version 115.3.130 and called the website again. Unfortunately the problem persists.

The blurry effect can be clearly seen here.
![2023-08-17 11_04_31-Window](https://github.com/cefsharp/CefSharp/assets/62557519/46ef4545-e731-4d93-9a61-166a79a2e9ce)

**- How does Chrome at the same version behave?**
I'm using Chrome version 115.0.5790.171 - but the problem doesn't exist here.

**- Have you tested on a different machine?**
Yes, i have.

**- How does the CEF Sample Application behave?**
The application displays the website correctly (with this Version https://cef-builds.spotifycdn.com/cef_binary_115.3.11%2Bga61da9b%2Bchromium-115.0.5790.114_windows32_client.tar.bz2)

**- What version are you using? (exact version please e.g. 75.1.143, not Latest version as that's relative)**
115.3.130

**- Are you using WPF/WinForms/OffScreen?**
WPF

**- Are you targeting .Net Core? If yes which version?**
.NET Framework 4.7.2

**Reply by amaitland** [reply]
Please confirm you tested `cefclient` with the following command line args as per https://github.com/cefsharp/CefSharp/blob/master/CONTRIBUTING.md#cefsharp-vs-chromium-embedded-frameworkcef

```
cefclient.exe --multi-threaded-message-loop --no-sandbox --off-screen-rendering-enabled --enable-gpu
```

What `DPI` is your system running at?

> **- Have you tested on a different machine?**
> Yes, i have.

Did the machine have different GPU?

**Comment by dongsiDaengsi** [comment]
Good morning.
The devices I tested it on have 300 dpi.
And yes, the devices have different GPUs. But the result is the same on each one: the blur

I also tested it with the Command Line command, unfortunately the same result.

**Reply by amaitland** [reply]
> I also tested it with the Command Line command, unfortunately the same result.

As the problem reproduces with `cefclient` then you'll need to raise an issue at https://github.com/chromiumembedded/cef/issues

**Reply by amaitland** [reply]
If you haven't already then please raise an issue at https://github.com/chromiumembedded/cef/issues (post a link back here for reference).
